src/table/head/table-head-cell.component.ts

```diff
@@ -17,7 +17,7 @@ import { TableHeaderItem } from "./../table-header-item.class";
 	template: `
 		<button
 			class="bx--table-sort"
-			*ngIf="sortable && !skeleton && this.sort.observers.length > 0 && column.sortable"
+			*ngIf="sortable && this.sort.observers.length > 0 && column.sortable"
```

Comment:
I _think_ the `!skeleton` was added to sort out a different skeleton bug ... maybe check the commit history for this to verify

Reply:
I think it was suppose to be removed but was added back by mistake in a future commit